README.md

```diff
@@ -41,147 +41,134 @@ _insert image of app_
-heroku-toolbelt/3.25.0 (x86_64-linux-gnu) ruby/2.1.5
-You have no installed plugins.
-</pre>
-</div>
+
+    $: lein -version
+    Leiningen 2.5.1 on Java 1.8.0_40-internal OpenJDK 64-Bit Server VM
+
+    $: git --version
+    git version 2.1.4
+
+    $: heroku --version
+    heroku-toolbelt/3.25.0 (x86_64-linux-gnu) ruby/2.1.5
+    You have no installed plugins.
 
 With the tools installed, we can begin working on our web app.
 
 ### Clojure, the Big Picture
 
-Clojure is a modern lisp with a focus on functional programming.
+Clojure is a modern Lisp with a focus on functional programming.
 
 <img src="http://griffsgraphs.files.wordpress.com/2012/07/programming-languages_label.png"/>
 
-There are lots of programming languages and Clojure is just one of
+There are lots of programming languages, and Clojure is just one of
 them.  Clojure is great because:
 
-+ The core language is small and easy to learn.
-+ The design makes it easier to write correct programs.
-+ Clojure makes it easier to write concurrent programs, programs that
+* The core language is small and easy to learn.
+* The design makes it easier to write correct programs.
+* Clojure makes it easier to write concurrent programs, programs that
 do more than one thing at a time.
-+ Clojure programs are fast.
-+ Clojure programs can build on Java libraries.
+* Clojure programs are fast.
+* Clojure programs can build on Java libraries.
 
 Most programmers have to use multiple languages to get their jobs
-done.  Web applications often use html, css, and javascript.  We'll
+done.  Web applications often use HTML, CSS, and JavaScript.  We'll
 touch on each of those as we build our web app.
 
 ### The Web
 
-The internet is a bunch of computer programs on computers all over the
-world sending messages to each other.  Some of those programs are
-servers, that listen for requests and respond with data.
+The Internet is a bunch computers all over the world communicating with
+each other using a variety of computer programs.  Some of those programs
+are servers that listen for requests and respond with data.
 
-Your browser is program that sends requests over "http".  Entering,
-[https://github.com](https://github.com) into your browser's address bar
-tells your browser that you want to see that page.
+Your browser is program that sends requests over HTTP (Hypertext
+Transfer Protocol).  Entering, [https://github.com](https://github.com)
+into your browser's address bar tells your browser that you want to see
+that page.
 
 [https://github.com](https://github.com) is actually a human-readable
 alias for the numerical address of GitHub's servers.  Since your
 computer isn't directly connected to GitHub, it asks the computers it
 is connected to to forward the request.
 
-On linux or the mac, ```traceroute``` will show you the number of hops it
-took to get to GitHub.  (On windows, the command is called ```tracert```.)
+On Linux or a Mac, `traceroute` will show you the number of hops a
```

Comment:
Three backticks tend to be for stand-alone code blocks, while single backticks are used for in-line bits of code formatting.